<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="SearchButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>90</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>search</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../Personal/Desktop/icon/search.png</normaloff>../../../Personal/Desktop/icon/search.png</iconset>
   </property>
  </widget>
  <widget class="QTreeView" name="treeView_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>231</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ChangePath">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Change</string>
   </property>
  </widget>
  <widget class="QComboBox" name="SearchComboBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Search Here</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Personal/Desktop/icon/..ico</normaloff>../../../Personal/Desktop/icon/..ico</iconset>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>231</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ResetButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BuildFile">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../Personal/Desktop/icon/folder.png</normaloff>../../../Personal/Desktop/icon/folder.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="SearchTips">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>90</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rename_box">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="RenameButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ReName</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../Personal/Desktop/icon/rename.png</normaloff>../../../Personal/Desktop/icon/rename.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../Personal/Desktop/icon/delete.png</normaloff>../../../Personal/Desktop/icon/delete.png</iconset>
   </property>
  </widget>
  <widget class="QTextBrowser" name="rename_background">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="rename_ok">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rename_cancel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CurrentPath">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <zorder>DeleteButton</zorder>
  <zorder>RenameButton</zorder>
  <zorder>BuildFile</zorder>
  <zorder>treeView_2</zorder>
  <zorder>ChangePath</zorder>
  <zorder>SearchComboBox</zorder>
  <zorder>tableWidget</zorder>
  <zorder>ResetButton</zorder>
  <zorder>SearchButton</zorder>
  <zorder>SearchTips</zorder>
  <zorder>rename_background</zorder>
  <zorder>rename_box</zorder>
  <zorder>rename_ok</zorder>
  <zorder>rename_cancel</zorder>
  <zorder>CurrentPath</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
